src/mame/brother/lw30.cpp

```diff
@@ -0,0 +1,1105 @@
+// license:BSD-3-Clause
+// copyright-holders:Bartman/Abyss
+
+#include "emu.h"
+#include "emupal.h"
+#include "screen.h"
+#include "speaker.h"
+#include "machine/timer.h"
+#include "cpu/z180/z180.h"
+#include "imagedev/floppy.h"
+#include "util/utf8.h"
+
+#include "sound/beep.h"
+#include "video/mc6845.h"
```

Comment:
Please organise these in the order prefix header, local, then most dependent to least dependent – it helps catch headers that don’t `#include` things they need.  The prefix header `emu.h` needs to come first, then the stuff from src/devices (keep them sorted aphabetically), then the stuff from src/emu, and then the stuff from lib/util.